trunk/NDHMS/Routing/module_HYDRO_io.F

```diff
@@ -8506,7 +8529,7 @@ subroutine readLinkSL( GNLINKSL,NLINKSL,route_link_f, route_lake_f, maxorder, &
                  CHLAT,      ZELEV,     TYPEL,    ORDER,    &
                  QLINK(:,1), MUSK,      MUSX,     CHANLEN,  &
                  MannN,      So,        ChSSlp,   Bw,       &
-                 Tw,         Tw_CC,     n_CC,     gages,    & 
+                 Tw,         Tw_CC,     n_CC,     ChannK, gages,    & 
```

Comment:
In this conflict, it appears that one of the calls to the subroutine read_route_link_netcdf within the subroutine readLinkSL was added later and did not include ChannK

Reply:
That is correct. I added ChannK to fix the issue.